deps/harfbuzz-icu-freetype

```diff
@@ -1 +1 @@
-Subproject commit 364b30bb26f3c4821e675f6b3027b4af74340ae4
+Subproject commit 1fe3240ba2b89a8cab9d7c4c586ecfbf66f18f03
```

Comment:
This got merged in the original project branch (https://github.com/tangrams/harfbuzz-icu-freetype/commits/remove-unused-deps), can you update this to: eccc706464b7b9858f44ffa779f3ba508ab648a7

Will merge this PR once this is done. Thanks